src/AppInstallerRepositoryCore/Rest/Schema/1_0/Json/ManifestDeserializer_1_0.cpp

```diff
@@ -366,7 +366,7 @@ namespace AppInstaller::Repository::Rest::Schema::V1_0::Json
         }
 
         // Installer Switches
-        installer.Switches = Manifest::GetDefaultKnownSwitches(installer.InstallerType);
+        installer.Switches = Manifest::GetDefaultKnownSwitches(installer.BaseInstallerType);

```

Comment:
Effective, unless it is not yet know, then some more effort will be required.

Reply:
Changed to effective